.circleci/config.yml

```diff
@@ -525,6 +525,7 @@ jobs:
             else
               export COMMIT_DOCKER_IMAGE=$output_image
             fi
+            echo "COMMIT_DOCKER_IMAGE: "${COMMIT_DOCKER_IMAGE}
```

Comment:
This is not needed, is it?
```suggestion
```

Reply:
It's needed if you want to look at the new image with a tool like dive. I removed it but I found it useful.

Reply:
We have this information when test is running, isn't it?

Reply:
Yes. You're right.